features/step_definitions/stepdefs.js

```diff
@@ -1184,6 +1186,65 @@ Then('{string} is not in our Followers', async function (actorName) {
     assert(!found);
 });
 
+Then('an Undo\\(Announce) is sent to {string}', async function (actorName) {
+    if (!this.actors[actorName]) {
+        throw new Error(`Could not find Actor ${actorName}`);
+    }
+    const actor = this.actors[actorName];
+
+    const inboxUrl = new URL(actor.inbox);
+
+    const foundInInbox = await waitForRequest(
+        'POST',
+        inboxUrl.pathname,
+        (call) => {
+            const body = JSON.parse(call.request.body);
+            return body.type === 'Undo' && body.object.type === 'Announce';
+        },
+    );
+
+    const foundActivity = JSON.parse(foundInInbox.request.body);
```

Comment:
Out of scope here I guess: but curious how would this test look like, if we wanted to stop relying on the inbox? Do we have currently another way to assert that an activity has been dispatched to the recipient(s) outbox(es)?

Reply:
This is checking that we're sending it to their inbox, so it doesn't depend on the KV inbox stuff - we're checking for a network request to an "external" inbox - does that make sense?

Reply:
Ah missed that! Great then :)